OperatingSystem-Services/Service-SOC/BasicDocumentation/Wazuh-Docker-Single-Node.md

```diff
@@ -0,0 +1,20 @@
+# How to Deploy Wazuh on Docker, Single-Node
```

Comment:
Maybe change the title to be shorter: `Docker - Single-Node Wazuh Deployment` Just a though you do no have to do so but making it shorter and easier to tell what it is at a glance.